app/models/header_lookup.rb

```diff
@@ -10,18 +10,22 @@ class HeaderLookup
   MESH_ID_URI = "https://id.nlm.nih.gov/mesh/"
   MEMOIZED_MESH_FILE = "memoized_mesh.txt"
   MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
+  MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
   SEARCHABLE_MESH_FILE = "subjects_mesh.yml"
   SEARCHABLE_LCSH_FILE = "subjects_lcsh.yml"
+  SEARCHABLE_LCNAF_FILE = "subjects_lcnaf.csv"
 
   def initialize
     puts "initializing header_lookup..."
     # these are the terms to search through for header lookups
     @@searchable_mesh_terms ||= YAML.load_file(SEARCHABLE_MESH_FILE)
     @@searchable_lcsh_terms ||= YAML.load_file(SEARCHABLE_LCSH_FILE)
+    @@searchable_lcnaf_file ||= CSV.new(SEARCHABLE_LCNAF_FILE)
```

Comment:
This creates the CSV object, but it doesn't actually read anything right away which is great. It might be worth doing this with the other searchable files to save some time on future exports.